kitchen.yml

```diff
@@ -31,18 +28,22 @@ suites:
   - name: server_install
     run_list:
       - recipe[test::server_install]
+    excludes: [amazonlinux-2]
```

Comment:
amazonlinux supports the postgresql client but not server, even its extras package supports client up to 11 but server up to 9.2

Reply:
If that's the case then I'm wondering if we should keep it supported since it makes it a one off. Let me take a closer look at this and get back to you.

Reply:
Actually, it looks like you need to fix something in ``libraries/helpers.rb`` [here](https://github.com/sous-chefs/postgresql/blob/master/libraries/helpers.rb#L229-L234).  Basically amazonlinux-2 is the same as centos-7 so if you change ``6`` to ``7`` it should work.